run.sh

```
@@ -0,0 +1,64 @@
+	  -exec rm {} +
+}
+
+# print all functions in this file
+function help {
+	echo "$0 <task> <args>"
+	echo "Tasks:"
+	compgen -A function | cat -n
+}
+
+TIMEFORMAT="Task completed in %3lR"
+time ${@:-help}
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)